x-pack/plugins/watcher/public/sections/watch_status/components/watch_status.tsx

```diff
@@ -4,18 +4,59 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React, { useEffect } from 'react';
-import { EuiPageContent, EuiSpacer } from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+import {
+  EuiPageContent,
+  EuiSpacer,
+  EuiTabs,
+  EuiTab,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiToolTip,
+  EuiBadge,
+  EuiButton,
+} from '@elastic/eui';
 import chrome from 'ui/chrome';
 import { MANAGEMENT_BREADCRUMB } from 'ui/management';
-
+import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
+import { toastNotifications } from 'ui/notify';
 import { WatchDetail } from './watch_detail';
 import { WatchHistory } from './watch_history';
 import { listBreadcrumb, statusBreadcrumb } from '../../../lib/breadcrumbs';
-import { loadWatchDetail } from '../../../lib/api';
+import { loadWatchDetail, deactivateWatch, activateWatch } from '../../../lib/api';
 import { WatchDetailsContext } from '../watch_details_context';
-import { getPageErrorCode, PageError, SectionLoading } from '../../../components';
+import {
+  getPageErrorCode,
+  PageError,
+  SectionLoading,
+  DeleteWatchesModal,
+} from '../../../components';
+import { goToWatchList } from '../../../lib/navigation';
+
+interface WatchStatusTab {
+  id: string;
+  name: string;
+}
+
+const WATCH_EXECUTION_HISTORY_TAB = 'watchExecutionHistoryTab';
+const WATCH_ACTIONS_TAB = 'watchActionsTab';
+
+const WATCH_STATUS_TABS: WatchStatusTab[] = [
+  {
+    id: WATCH_EXECUTION_HISTORY_TAB,
+    name: i18n.translate('xpack.watcher.sections.watchStatus.executionHistoryTabLabel', {
+      defaultMessage: 'Execution history',
+    }),
+  },
+  {
+    id: WATCH_ACTIONS_TAB,
+    name: i18n.translate('xpack.watcher.sections.watchStatus.actionsTabLabel', {
+      defaultMessage: 'Actions',
```

Comment:
What do you think about changing this tab to read 'Action statuses' or 'Action status'. The extra word seems to provide a fair bit of clarity, imo.

Reply:
Good idea, updated tab to 'Action statuses'